src/System.Numerics.Tensors/src/System.Numerics.Tensors.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <Configurations>netstandard-Debug;netstandard-Release;netstandard1.1-Debug;netstandard1.1-Release;</Configurations>
+    <DocumentationFile>$(OutputPath)$(MSBuildProjectName).xml</DocumentationFile>
+    <RootNamespace>System.Numerics.Tensors</RootNamespace>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="Properties\InternalsVisibleTo.cs" />
+    <Compile Include="System\Numerics\Tensors\ArrayTensorExtensions.cs" />
+    <Compile Include="System\Numerics\Tensors\ArrayUtilities.cs" />
+    <Compile Include="System\Numerics\Tensors\CompressedSparseTensor.cs" />
+    <Compile Include="System\Numerics\Tensors\DenseTensor.cs" />
+    <Compile Include="System\Numerics\Tensors\SparseTensor.cs" />
+    <Compile Include="System\Numerics\Tensors\Tensor.cs" />
+    <Compile Include="System\Numerics\Tensors\TensorView.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <Reference Include="System.Memory" />
```

Comment:
This is a bit tricky since this is an OOB package. We don't want to become dependent on any new APIs from `System.Memory`. We only want to target the `4.5.1` version.

@weshaggard @ericstj - any thoughts on how to only compile against a specific version of a NuGet package from corefx?

Reply:
So long as the reference assembly version isn't updated that's what it will depend on.  PackageIndex controls this. (not to mention we don't even build System.Memory package anymore).  Please build the package locally and examine the dependencies.